gcs/src/test/java/com/google/cloud/hadoop/fs/gcs/contract/GoogleContract.java

```diff
@@ -36,8 +36,6 @@ public GoogleContract(Configuration conf, TestBucketHelper bucketHelper) {
     super(conf);
     addConfResource(CONTRACT_XML);
     conf.set("fs.contract.test.fs.gs", "gs://" + bucketHelper.getUniqueBucketPrefix());
-    // TODO: remove when GCS fixes non-retriable "410 Gone" responses
-    conf.setInt("fs.gs.outputstream.upload.cache.size", 1024 * 1024);
```

Comment:
Is this a dup change as another PR?

Reply:
Yes, there are the same commits, I will rebase this PR when previous will be merged.